code42_connector.py

```diff
@@ -361,6 +362,124 @@ def _handle_remove_legal_hold_custodian(self, param, action_result):
         status_message = f"{username} was removed from legal hold matter {matter_id}."
         return action_result.set_status(phantom.APP_SUCCESS, status_message)
 
+    @action_handler_for("create_case")
+    def _handle_create_case(self, param, action_result):
+        name = param["case_name"]
+        description = param.get("description")
+        subject = param.get("subject")
+        if subject is not None:
+            subject = self._get_user_id(subject)
+        assignee = param.get("assignee")
+        if assignee is not None:
+            assignee = self._get_user_id(assignee)
+        findings = param.get("findings")
+        response = self._client.cases.create(
+            name,
+            description=description,
+            subject=subject,
+            assignee=assignee,
+            findings=findings,
+        )
+        case_number = response["number"]
+        action_result.add_data(response.data)
+        status_message = f"Case successfully created with case_id: {case_number}"
+        action_result.update_summary(
```

Comment:
Seems like a lot of info for the summary

Reply:
trimmed these down to just the case number